json_annotation/lib/src/json_key.dart

```diff
@@ -8,7 +8,7 @@ import 'allowed_keys_helpers.dart';
 import 'json_serializable.dart';
 
 /// An annotation used to specify how a field is serialized.
```

Comment:
Expand the doc comment here to say that it's valid for both properties and ctor params.